### ISSUE TITLE
HTML validation errors

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2375]
On current master demo content (0.8  just released):

1
Error: Attribute label not allowed on element button at this point.
`<button type="button" aria-expanded="true" label="Open section: Status &amp; Visibility" ...`

this is because the panel body Button receives a `label` prop that should be `aria-label`

2
Error: Element figcaption not allowed as child of element div in this context. 
```
<div class="blocks-editable"><figcaption contenteditable="true" class="blocks-editable__tinymce mce-content-body" aria-label="Write caption…" id="mce_7" data-is-placeholder-visible="false">
```

3
Error: Element div not allowed as child of element span in this context.
`<div class="editor-visual-editor__block-edit"><span class="wp-block-button"><div class="blocks-editable">`

4
Error: The scrolling attribute on the iframe element is obsolete. Use CSS instead.

**Comment by steffiau** [comment]
I'd like to work on this. :)

**Comment by youknowriad** [comment]
@steffiau Go for it 👍

**Comment by steffiau** [comment]
Forgot I don't have a server anymore for my wordpress sites, I'm setting up a local wordpress environment but this might take me a while. If anyone else wants to pick up this issue instead, please feel free by all means!

**Comment by Xyfi** [comment]
I've ran the demo content page through validtor.w3.org, and no errors or warnings were shown. Is the still relevant?

**Comment by afercia** [comment]
Hm this related to the admin Gutenberg page, just tried again and passing the HTML starting from `<div class="gutenberg">` through https://validator.w3.org/nu/ I get a total of 31 errors + warnings

**Comment by talldan** [comment]
I noticed a couple:

1. The RichText adds an invalid `aria-expanded` attribute to `p` tags. Notably in the paragraph block. 
2. The BlockIcon component renders a `div`, but is used inside `button` elements. 

It might be worth having separate issues instead of a single umbrella issue.

**Comment by tofumatt** [comment]
I'm not sure the original issue is clear enough to work on. What are the steps to reproduce here and what is the solution? I'm finding this one a bit clear and, as @talldan mentioned, possibly containing more than one issue 😄
